code/modules/mob/say.dm

```diff
@@ -101,6 +101,19 @@
 ///The amount of items we are looking for in the message
 #define MESSAGE_MODS_LENGTH 6
 
+/mob/proc/check_for_custom_say_emote(message, list/mods)
```

Comment:
add a comment for what this does as well, it's a bit confusing

Reply:
Done to the best of my current knowledge of it, might not be a useful comment but I tried my best.